cloud_blobstore/__init__.py

```diff
@@ -91,7 +97,8 @@ def list_v2(
             delimiter: str=None,
             start_after_key: str=None,
             token: str=None,
-            k_page_max: int=None
+            k_page_max: int=None,
+            verbose: bool=None,
```

Comment:
I think `verbose` dose not accurately describe what's going on here.

We should never expose the cloud-native blob handle to the caller.  Furthermore, there's almost no advantage to having this `verbose` differentiation because the data is being fetched _anyway_.

Here's my proposal: make a breaking change to the API.

- list_v2 returns an iterator always returns a list of tuples.  first element is the key name, second element is a dict of {property_name: property}.  property_name is normalized across cloud providers, which means cloud-blobstore needs to declare a set of properties it will transform from each cloud provider's vocabulary.

A possibly less drastic proposal would be to keep the verbose/not-verbose distinction (and thus not introduce an API breaking change), but still return the property map as the second element of the tuple rather than the cloud-native blob handle.

Reply:
That sounds like a good idea. I considered the advantages of returning a consistent data structure as opposed to the cloud-native blob, but thought another layer of abstraction from the existing, well-documented blob structures would be too confusing. However, I see now that never exposing the cloud-native blobs in this library is a better solution. Is there documentation for the `cloud-blobstore` API functions somewhere? If not, I'd be willing to create that.

I'll go with the non API breaking change.

Reply:
No, it's just in the docblocks.

Reply:
In lieu of a parameter that changes the returned type, and since the data is fetched anyway, I'd prefer the suggestion from @ttung to break the API.